unit_tests/speech/test_client.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import base64
+
+from google.cloud._helpers import _to_bytes
+
+_AUDIO_CONTENT = _to_bytes('/9j/4QNURXhpZgAASUkq')
+_B64_AUDIO_CONTENT = base64.b64encode(_AUDIO_CONTENT)
+
+
+class TestClient(unittest.TestCase):
+    SAMPLE_RATE = 16000
+    HINTS = ["hi"]
+
+    def _getTargetClass(self):
+        from google.cloud.speech.client import Client
+        return Client
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test_ctor(self):
+        from google.cloud.speech.connection import Connection
+
+        creds = _Credentials()
+        http = object()
+        client = self._makeOne(credentials=creds, http=http)
+        self.assertTrue(isinstance(client.connection, Connection))
```

Comment:
You could / should use `self.assertIsInstance`. This method was absent in Python 2.6, which we used to support, which is why you find it sprinkled in some of our tests.